cookieConsent/OneTrust/browser.js

```diff
@@ -0,0 +1,34 @@
+class OneTrust {
+  constructor(sourceConfig) {
+    this.sourceConfig = sourceConfig;
+  }
+
+  isEnabled(destConfig) {
+    const { oneTrustConsentGroup } = destConfig; // mapping of the destination with the consent group name
+    const userSetConsentGroupIds = window.OnetrustActiveGroups.split(","); // Content ids user has rejected
+    const oneTrustAllGroupsInfo = window.OneTrust.GetDomainData().Groups; // info about all the groups created in the onetrust account
```

Comment:
Add error handling around this logic as you cannot always guarantee the user would do the right thing

Reply:
yes will do the error handlings.

Reply:
done.

Reply:
Add try-catch blocks around the core logic

Reply:
done